app/src/main/res/drawable/thumbnail_gradient.xml

```diff
@@ -4,6 +4,6 @@
   <gradient
     android:angle="270"
     android:startColor="#00FFFFFF"
```

Comment:
Please create a similar value for startColor too

Reply:
> Please create a similar value for startColor too

Actually startColor not changing in dark_mode it's same. Still I have to make value?